src/main/scala/devices/debug/DebugTransport.scala

```diff
@@ -65,24 +65,29 @@ class DTMInfo extends Bundle {
 /** A wrapper around JTAG providing a reset signal and manufacturer id. */
 class SystemJTAGIO extends Bundle {
   val jtag = Flipped(new JTAGIO(hasTRSTn = false))
-  val reset = Input(Bool())
+  val reset = Input(Reset())
```

Comment:
is there ever a case where we wouldn't expect this to be `AsyncReset`? Should we just set it that way...?

Reply:
Done.